src/lib/Gen/adoc/AdocGenerator.cpp

```diff
@@ -69,10 +76,35 @@ build(
     if(! ex)
         return ex.error();
 
-    MultiPageVisitor visitor(*ex, outputPath, corpus);
+    std::string path = files::appendPath(outputPath, "reference.tag.xml");
```

Comment:
We need an option here, such as:

https://www.doxygen.nl/manual/config.html#cfg_generate_tagfile

Its default value can be `"reference.tag.xml"` or whatever and tagfiles won't be generated when it's empty.